packages/dds/tree/src/shared-tree-core/transaction.ts

```diff
@@ -92,12 +92,29 @@ export interface TransactionEvents {
 	committing(): void;
 }
 
+/**
+ * Callbacks for transaction lifecycle events.
+ */
+export interface Callbacks {
+	/**
+	 * Called when the current transaction is popped from the {@link TransactionStack | stack}.
+	 */
+	readonly onPop?: OnPop;
+	/**
+	 * The function to call when a nested transaction is next pushed onto the {@link TransactionStack | stack}.
```

Comment:
The doc comment for `onPop` is written as if the audience is a receiver of a `Callbacks` ("called when..."), but the doc comment for `onPush` is written as if the audience is a creator/owner of a `Callbacks` ("the function to call"). I'm not sure which audience we should address these docs to, but probably not one of each.